src/fw/drivers/pmic/npm1300.c

```diff
@@ -176,16 +183,21 @@ bool pmic_init(void) {
 }
 
 bool pmic_power_off(void) {
-  /* TODO(FIRM-114): shut almost everything off, but not the nRF Vdd, and then put the nRF in system_off */
-  return false;
+  if (pmic_is_usb_connected()) {
+    PBL_LOG(LOG_LEVEL_ERROR, "USB is connected, cannot power off");
+    return false;
+  }
+
+  if (!prv_write_register(PmicRegisters_SHIP_TASKENTERSHIPMODE, 1)) {
+    PBL_LOG(LOG_LEVEL_ERROR, "Failed to enter ship mode");
+    return false;
+  }
+
+  return true;
 }
 
-// This is a hard power off, resulting in all rails being disabled.
-// Generally, this is not desirable since we'll lose the backup domain.
-// You're *probably* looking for pmic_power_off.
 bool pmic_full_power_off(void) {
-  /* TODO(FIRM-114): put the PMIC in ship mode */
-  return false;
+  return pmic_power_off();
 }
```

Comment:
OK with me for now, but please file a GitHub issue to differentiate these later.

Reply:
#238